cfme/tests/openstack/cloud/test_networks.py

```diff
@@ -123,6 +123,8 @@ def test_edit_network(network):
     wait_for(network.provider_obj.is_refreshed, func_kwargs=dict(refresh_delta=10), timeout=600,
              delay=10)
     network.browser.refresh()
```

Comment:
Required:
No need of this as you are using `wait_for` and refreshing `browser`

Reply:
Thanks. I agree that I should delete refreshing browser. But without this `wait_for` this test is failed.

Reply:
I am did not talk about removing `wait_for`

Reply:
fixed